centreon/www/front_src/src/Resources/Filter/criteriasNewInterface/basicFilter/atoms.ts

```diff
@@ -0,0 +1,13 @@
+import { atomWithStorage } from 'jotai/utils';
+import { atom } from 'jotai';
+
+import { SelectedResourceType } from '../model';
+
+export const selectedStatusByResourceTypeAtom =
+  atomWithStorage<Array<SelectedResourceType> | null>(
+    'FilterSelectedStatus',
```

Comment:
I think you should contextualize the key (for instance: centreon-{{version}}-resourceStatusRevamp-filterSelecedStatus